### PR TITLE
Teamcard glowup

### PR DESCRIPTION
**Pull request by frzyc**
## Describe your changes

Making some changes according to figma
image

## Issue or discord link

- 

## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)

- [ ] I have commented my code in hard-to understand areas.
- [ ] I have made corresponding changes to README or wiki.
- [ ] For front-end changes, I have updated the corresponding English translations.
- [ ] I have run `yarn run mini-ci` locally to validate format and lint.
- [ ] If I have added a new library or app, I have updated the deployment scripts to ignore changes as needed

### CODE REVIEW

File: libs/common/ui/src/components/Card/CardThemed.tsx [hunk 1]
```diff
@@ -18,5 +18,7 @@ export const CardThemed = styled(Card, {
       ? theme.palette.contentLight.main
```
Comment (nguyentvan7) [inline libs/common/ui/src/components/Card/CardThemed.tsx:18]: could you make this a separate function with if else/switch case and return statements? nested  nested nested ternary is impossible to read

File: libs/gi/ui/src/components/team/TeamCard.tsx [hunk 2]
```diff
@@ -357,15 +411,236 @@ function CharacterArea({
...
+                  <Typography>
+                    <ArtifactSlotName slotKey="sands" />
+                  </Typography>
+                  <Typography>{t(statKey)}</Typography>
+                </Box>
+              }
+            >
+              <Typography sx={{ lineHeight: 0 }}>
+                {slotKey === 'sands' && (
+                  <SandsIcon sx={{ fontSize: 'inherit' }} />
```
Comment (nguyentvan7) [inline libs/gi/ui/src/components/team/TeamCard.tsx:529]: I hovered over this and crashed

Please enter the steps to reproduce this error below:
1. 

**Don't forget to upload your JSON file!**

```
Message: Minified React error #185; visit https://reactjs.org/docs/error-decoder.html?invariant=185 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
Name: Error
Stack: Error: Minified React error #185; visit https://reactjs.org/docs/error-decoder.html?invariant=185 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at du (https://genshin-optimizer-prs.github.io/pr/2323/frontend/assets/index-C6KbjYFe.js:48:33862)
    at efe (https://genshin-optimizer-prs.github.io/pr/2323/frontend/assets/index-C6KbjYFe.js:46:22379)
    at https://genshin-optimizer-prs.github.io/pr/2323/frontend/assets/index-C6KbjYFe.js:131:23586
    at Ty (https://genshin-optimizer-prs.github.io/pr/2323/frontend/assets/index-C6KbjYFe.js:48:24296)
    at Yx (https://genshin-optimizer-prs.github.io/pr/2323/frontend/assets/index-C6KbjYFe.js:48:31641)
    at IV (https://genshin-optimizer-prs.github.io/pr/2323/frontend/assets/index-C6KbjYFe.js:48:31497)
    at IV (https://genshin-optimizer-prs.github.io/pr/2323/frontend/assets/index-C6KbjYFe.js:48:31405)
    at IV (https://genshin-optimizer-prs.github.io/pr/2323/frontend/assets/index-C6KbjYFe.js:48:31405)
    at IV (https://genshin-optimizer-prs.github.io/pr/2323/frontend/assets/index-C6KbjYFe.js:48:31405)
    at IV (https://genshin-optimizer-prs.github.io/pr/2323/frontend/assets/index-C6KbjYFe.js:48:31405)
```

**Review by nguyentvan7 — CHANGES_REQUESTED** [review]

**Review by nguyentvan7 — CHANGES_REQUESTED** [review]
Screenshot_20240718_220615_Chrome

Hover always says "sands"? (This is a circlet)

**Comment by frzyc** [comment]
Good catch.

**Review by nguyentvan7 — APPROVED** [review]

**Pull request merged**